Automattic/newspack-popups
# feat: deprecate "display title" option

### All Submissions:






### Changes proposed in this Pull Request:



Deprecates the little-used "Display Prompt Title" option.

Closes `1204323000227080/1204082722103227`.

### How to test the changes in this Pull Request:

1. On `master`, publish a prompt with the "Display Prompt Title" option enabled and observe that the title is displayed when the prompt is rendered on the front-end.
2. Check out this branch and confirm that the option is gone in the prompt editor, and the title no longer renders on the front-end.

### Other information:
